commit 1bfa17d69f62a7bda3a2e419349692f36f3776db
Author: Josh Cooper <joshcooper@users.noreply.github.com>
Date:   Tue Nov 15 11:37:53 2022 -0800

    Solaris 10 SPARC patch for Ruby 2.7
    
    This is the same patch as [1] but it's adapted to Ruby 2.7
    
    [1] resource/patches/ruby_25/Solaris-only-Replace-reference-to-RUBY-var-with-opt-pl-build-tool.patch

diff --git a/ext/etc/depend b/ext/etc/depend
index 99e812c7e4..8c9f926d36 100644
--- a/ext/etc/depend
+++ b/ext/etc/depend
@@ -1,6 +1,6 @@
 constdefs.h : $(srcdir)/mkconstants.rb
 	@echo "generating constant definitions"
-	@$(RUBY) $(srcdir)/mkconstants.rb -o constdefs.h
+	@/opt/csw/bin/ruby $(srcdir)/mkconstants.rb -o constdefs.h
 
 # AUTOGENERATED DEPENDENCIES START
 etc.o: $(RUBY_EXTCONF_H)
diff --git a/ext/ripper/depend b/ext/ripper/depend
index 69759ec716..3c1314186a 100644
--- a/ext/ripper/depend
+++ b/ext/ripper/depend
@@ -29,7 +29,7 @@ check: .eventids2-check
 
 .eventids2-check: $(GEN) $(SRC1) $(SRC2)
 	$(ECHO) checking $(SRC1) and $(SRC2)
-	$(Q) $(RUBY) $(GEN) --mode=check --ids1src=$(SRC1) --ids2src=$(SRC2)
+	$(Q) /opt/csw/bin/ruby $(GEN) --mode=check --ids1src=$(SRC1) --ids2src=$(SRC2)
 	@exit > $@
 
 eventids1.c: $(GEN) $(srcdir)/tools/dsl.rb $(SRC1)
